### PR TITLE
3403 Bulkdownload Repository files

### LINKED ISSUE #3403: Bulk download for the references stored in the repo
@minotogna 
**Is your feature request related to a problem? Please describe.**
There might be many files uploaded to the repository. It would be convenient to add an additional function to download all the files such as a zip file at once.

**Describe the solution you'd like**
Add a button called 'Bulk Download.'
image

### PR DESCRIPTION
**Pull request by sorja**
https://github.com/openforis/fra-platform/assets/5508251/2192767b-8ed6-4187-b753-7f14bffcec6e

Global files:
image

Country files: (From video, Finland, demo)
image

### CODE REVIEW

File: src/client/pages/CountryHome/Repository/Repository.scss [hunk 1]
```diff
@@ -12,10 +12,11 @@
   align-items: center;
   display: flex;
   height: $spacing-s;
-  justify-content: center;
-  margin-top: $spacing-s;
+  justify-content: flex-start;
+  margin: $spacing-m 0 $spacing-xs;
```
Comment (minotogna) [inline src/client/pages/CountryHome/Repository/Repository.scss]: Why different margins top and bottom ? I would make the button download size s

File: src/meta/api/endpoint/ApiEndPoint.ts [hunk 2]
```diff
@@ -72,7 +72,10 @@ export const ApiEndPoint = {
     },
 
     Repository: {
+      // TODO: Move to Repository.File
```
Comment (minotogna) [inline src/meta/api/endpoint/ApiEndPoint.ts]: What's this Todo ?

Comment (sorja) [reply src/meta/api/endpoint/ApiEndPoint.ts]: Should we move file related endpoints to `...Repository.File.{one,many}`?

Comment (minotogna) [reply src/meta/api/endpoint/ApiEndPoint.ts]: Sure . What do u think ?

File: src/server/controller/cycleData/repository/index.ts [hunk 3]
```diff
@@ -3,15 +3,17 @@ import { RepositoryRepository } from 'server/repository/assessmentCycle/reposito
...
 import { update } from './update'
 
 export const Repository = {
   create,
-  getMany: RepositoryRepository.getMany,
-  getOne: RepositoryRepository.getOne,
   getFile,
   getFileMeta,
-  update,
+  getMany: RepositoryRepository.getMany,
```
Comment (minotogna) [inline src/server/controller/cycleData/repository/index.ts:14]: Should get file be renamed to get one file ?

File: src/meta/api/endpoint/ApiEndPoint.ts (hunk 2 above)
Comment (sorja) [reply src/meta/api/endpoint/ApiEndPoint.ts]: sure

**Review by minotogna — APPROVED** [review]

**Pull request merged**